sw/device/lib/dif/dif_keymgr.c

```diff
@@ -0,0 +1,9 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_keymgr.h"
+
+#include "keymgr_regs.h"  // Generated.
+
+#error "Key Manager DIF library is not implemented yet."
```

Comment:
I would drop the #error, since it makes it so this library can't be built. I think it's fine to just have an empty file so that the header is built and checked.

Reply:
This is based on @lenary's comment [here](https://github.com/lowRISC/opentitan/pull/3142#discussion_r493444840). I don't have a strong preference either way.